packages/jest-watch/src/jest_hooks.js

```diff
@@ -61,11 +61,11 @@ class JestHooks {
         this._listeners.onTestRunComplete.forEach(listener =>
           listener(results),
         ),
-      shouldRunTestSuite: async testPath =>
+      shouldRunTestSuite: async (testPath, config) =>
```

Comment:
Another option for this would be to change the signature to `({ testPath, config }) => {}`, which might be better for future API changes, given that adding keys to that object would not be a breaking change.

Reply:
Although this would mean an initial breaking change, which should probably only affect `jest-watch-typeahead`

Reply:
I hope people will understand it's still getting in shape ;)

Reply:
We can call it out in the blog post as still somewhat experimental?